bom/camunda-only-bom/pom.xml

```diff
@@ -209,6 +209,29 @@
         <artifactId>camunda-spin-bom</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.camunda.spin</groupId>
+        <artifactId>camunda-spin-dataformat-all</artifactId>
+        <version>${project.version}</version>
+        <exclusions>
```

Comment:
We don't have an other `<exclusions>` in `bom/camunda-only-bom/pom.xml` dependencyManagement.
I think it is cleaner and safer then copying the `<exclusions>`-s to the other 5 places where `camunda-spin-dataformat-all` is used.

Reply:
👍 Sounds good to me